src/views/ImportFlow.vue

```diff
@@ -285,4 +285,9 @@ export default ImportFlow
   overflow: hidden;
   white-space: nowrap;
 }
+
+.card {
```

Comment:
Quick fix to ensure that this is always display flex as it is in the community builder so that when you remove the import-note from messages.json you still get the same width card.

Reply:
Can we use the bootstrap `d-flex` class instead?